bitnami/redis/values.yaml

```diff
@@ -1816,6 +1826,16 @@ metrics:
     ## @param metrics.podMonitor.targetLimit Limit of how many targets should be scraped
     ##
     targetLimit: false
+    ## @param metrics.podMonitor.additionalEndpoints  Additional endpoints to scrape (e.g sentinel)
+    ##
+    additionalEndpoints: []
+      # uncomment in order to scrape sentinel metrics
```

Comment:
there is no port named http-metrics, it only valid when using ServiceMonitoer & metrics-svc, i'll  fix this comment